rest/rest-cli/src/main/resources/org/ow2/proactive_grid_cloud_portal/cli/cmd/commands.js

```diff
@@ -249,16 +251,53 @@ function listjobs(x, y) {
     }
 }
 
+// expected params: jobId, tag, offset, limit 
+function listtasks(arg1, arg2, arg3, arg4) {
```

Comment:
There are 4 different ways to list tasks:
* by jobId -> arg1
* by joId + tag -> arg1, arg2
* by jobId and pagination -> arg1, arg2, arg3
* by jobId + tag + pagination -> arg1, arg2, arg3, arg4
I actually need 4 functions but there's no function overloading in javascript afaik, thus the need for generic names on parameters.